webpack.dev.js

```diff
@@ -6,6 +6,17 @@ const webpack = require('webpack');
 
 module.exports = merge(common, {
     mode: 'development',
+    watchOptions: {
+        // Since we use require.context, webpack is watching the entire directory.
+        // We need to exclude any files we don't want webpack to watch.
+        // See: https://webpack.js.org/configuration/watch/#watchoptions-exclude
+        ignored: [
+            '**/{node_modules,dist,docs,e2e}', // All files in node_modules, dist, docs, e2e,
```

Comment:
Note: this may need to be modified when the playwright team implements watch mode